src/layouts/Header/Dropdown.jsx

```diff
@@ -14,6 +14,8 @@ function Dropdown({ currentUser }) {
   const { provider } = useParams()
   const isGh = providerToName(provider) === 'Github'
 
+  const to = `${window.location.protocol}//${window.location.host}/login`
```

Comment:
Is there a reason you're accessing the window directly instead of using useLocation? I'm pretty sure its a proxy of window but it would make testing/mocking this easier then a global in the spec.